tools/src/main/java/org/apache/kafka/trogdor/common/JsonUtil.java

```diff
@@ -53,4 +57,41 @@ public static String toPrettyJsonString(Object object) {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * Determine if a string is a JSON object literal.
```

Comment:
Do we have any use cases for lists of objects at the top level of a JSON string?

Reply:
That's a fair question.  The answer here is that in all the cases where we're using this function, we want a JSON object as the top-level thing.  We generally always make things objects in case we need to add more stuff later on.  (For example, perhaps right now we want to configure X with a list of ints, but later we might want a list of ints and a float, etc.)